recipes/synced_collections/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "synced_collections" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "b770a99c8455cb00dce4645525f8092f4ec15a9d028276777f266f9652f6b8f5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/glotzerlab/synced_collections/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - setuptools
+  run:
+    - python >=3.8
+
+test:
+  imports:
+    - synced_collections
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://signac.io
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: Interact with persistent key-value stores using Pythonic abstractions.
+  description: |
+    The signac framework helps users manage and scale file-based workflows,
+    facilitating data reuse, sharing, and reproducibility. It provides a simple
+    and robust data model to create a well-defined indexable storage layout for
+    data and metadata. This makes it easier to operate on large data spaces,
+    streamlines post-processing and analysis and makes data collectively
+    accessible.
```

Comment:
Can we rewrite this? Proposing this:

> The synced_collections package provides Pythonic abstractions over various underlying data stores, presenting APIs that behave like standard built-in Python collections like dicts.

Reply:
Yes I'm fine with rewriting. Happy to use something like this.